kit/layout/src/Skeleton/Skeleton.tsx

```diff
@@ -193,18 +199,21 @@ export function UISkeleton({
 
     const { isVisible, crossDissolveProgress } = useCrossDissolve(visible);
 
+    const { borderRadius: stylePropBorderRadius } = StyleSheet.flatten(styleProp);
+    const skeletonBorderRadius =
+        stylePropBorderRadius !== undefined
+            ? stylePropBorderRadius
+            : UILayoutConstant.alertBorderRadius;
+
     return (
-        <View
-            style={[
-                styles.container,
-                { borderRadius: isVisible ? UILayoutConstant.alertBorderRadius : 0 },
-                styleProp,
-            ]}
-            onLayout={onLayout}
-        >
-            {children}
+        <View style={[styles.container, styleProp]} onLayout={onLayout}>
+            <View style={{ opacity: show ? 0 : 1 }}>{children}</View>
```

Comment:
Does it mean that `Skeleton` isn't behave like a regular `View` anymore? For example setting `flexDirection: row` for it won't have any effect on content.

Reply:
You're right, I'm going back to the previous solution with a little revision